tests/platform_tests/raw-messages

```diff
@@ -0,0 +1,636 @@
+
```

Comment:
Is this file necessary?

Reply:
Removed it now. I added to test the test - reduce time taken to run the entire test by directly jumping to log_analyzer.
But, this isn't necessarily useful.